site/content/en/docs/user/kwokctl-snapshot/yaml-snapshot.demo

```diff
@@ -0,0 +1,29 @@
+# Let's getting started with kwokctl!
+kwokctl create cluster
+
+# Create an nodes.
+kwokctl scale node --replicas 2
+
+# Apply a deployment.
```

Comment:
The same suggestion in the other review about using a context first applies here. So, the below applies before creating a deployment.

```suggestion
# Use context.
kubectl cluster-info --context kwok-kwok

# Apply a deployment.
```